### ISSUE TITLE
Unexpected MAE training loss

### ISSUE THREAD

**Issue by liming-ai** [open-mmlab/mmselfsup#283]
@YuanLiuuuuuu 

I use the same [config](https://github.com/open-mmlab/mmselfsup/blob/master/configs/selfsup/mae/mae_vit-base-p16_8xb512-coslr-400e_in1k.py) but replace `imgs_per_gpu` with `samples_per_gpu`, however, the loss of MAE decreases rapidly, and then almost unchanged.

---

My detailed config:
```
model = dict(
    type='MAE',
    backbone=dict(type='MAEViT', arch='b', patch_size=16, mask_ratio=0.75),
    neck=dict(
        type='MAEPretrainDecoder',
        patch_size=16,
        in_chans=3,
        embed_dim=768,
        decoder_embed_dim=512,
        decoder_depth=8,
        decoder_num_heads=16,
        mlp_ratio=4.0),
    head=dict(type='MAEPretrainHead', norm_pix=True, patch_size=16))
data_source = 'ImageNet'
dataset_type = 'SingleViewDataset'
img_norm_cfg = dict(mean=[0.485, 0.456, 0.406], std=[0.229, 0.224, 0.225])
train_pipeline = [
    dict(
        type='RandomResizedCrop', size=224, scale=(0.2, 1.0), interpolation=3),
    dict(type='RandomHorizontalFlip'),
    dict(type='ToTensor'),
    dict(
        type='Normalize',
        mean=[0.485, 0.456, 0.406],
        std=[0.229, 0.224, 0.225])
]
prefetch = True
data = dict(
    samples_per_gpu=512,
    workers_per_gpu=16,
    train=dict(
        type='SingleViewDataset',
        data_source=dict(
            type='ImageNet',
            data_prefix='data/imagenet/train',
            ann_file='data/imagenet/meta/train.txt'),
        pipeline=[
            dict(
                type='RandomResizedCrop',
                size=224,
                scale=(0.2, 1.0),
                interpolation=3),
            dict(type='RandomHorizontalFlip'),
            dict(type='ToTensor'),
            dict(
                type='Normalize',
                mean=[0.485, 0.456, 0.406],
                std=[0.229, 0.224, 0.225])
        ],
        prefetch=False))
optimizer = dict(
    type='AdamW',
    lr=0.0024,
    betas=(0.9, 0.95),
    weight_decay=0.05,
    paramwise_options=dict(
        norm=dict(weight_decay=0.0),
        bias=dict(weight_decay=0.0),
        pos_embed=dict(weight_decay=0.0),
        mask_token=dict(weight_decay=0.0),
        cls_token=dict(weight_decay=0.0)))
optimizer_config = dict()
lr_config = dict(
    policy='StepFixCosineAnnealing',
    min_lr=0.0,
    warmup='linear',
    warmup_iters=40,
    warmup_ratio=0.0001,
    warmup_by_epoch=True,
    by_epoch=False)
runner = dict(type='EpochBasedRunner', max_epochs=100)
checkpoint_config = dict(interval=1, max_keep_ckpts=3, out_dir='')
log_config = dict(interval=100, hooks=[dict(type='TextLoggerHook')])
dist_params = dict(backend='nccl')
cudnn_benchmark = True
log_level = 'INFO'
load_from = None
resume_from = None
workflow = [('train', 1)]
persistent_workers = True
opencv_num_threads = 0
```
---
The training log
```
Epoch [1][100/313]	lr: 4.942e-06, eta: 10:30:10, time: 1.212, data_time: 0.266, memory: 66330, loss: 1.0658
Epoch [1][200/313]	lr: 1.015e-05, eta: 9:07:33, time: 0.901, data_time: 0.002, memory: 66330, loss: 0.2416 
Epoch [1][300/313]	lr: 1.565e-05, eta: 8:37:25, time: 0.892, data_time: 0.001, memory: 66330, loss: 0.0463
Epoch [2][100/313]	lr: 2.221e-05, eta: 8:30:20, time: 1.090, data_time: 0.148, memory: 66330, loss: 0.0060
Epoch [2][200/313]	lr: 2.833e-05, eta: 8:19:27, time: 0.899, data_time: 0.001, memory: 66330, loss: 0.0058
Epoch [2][300/313]	lr: 3.474e-05, eta: 8:11:30, time: 0.898, data_time: 0.002, memory: 66330, loss: 0.0058
Epoch [3][100/313]	lr: 4.235e-05, eta: 8:09:20, time: 1.081, data_time: 0.152, memory: 66330, loss: 0.0058
Epoch [3][200/313]	lr: 4.940e-05, eta: 8:04:16, time: 0.904, data_time: 0.002, memory: 66330, loss: 0.0058
Epoch [3][300/313]	lr: 5.676e-05, eta: 7:59:29, time: 0.895, data_time: 0.001, memory: 66330, loss: 0.0058
Epoch [4][100/313]	lr: 6.545e-05, eta: 7:57:08, time: 1.059, data_time: 0.147, memory: 66330, loss: 0.0058
Epoch [4][200/313]	lr: 7.347e-05, eta: 7:53:37, time: 0.902, data_time: 0.001, memory: 66330, loss: 0.0058
Epoch [4][300/313]	lr: 8.180e-05, eta: 7:50:13, time: 0.897, data_time: 0.002, memory: 66330, loss: 0.0058
Epoch [5][100/313]	lr: 9.160e-05, eta: 7:48:28, time: 1.061, data_time: 0.139, memory: 66591, loss: 0.0058
Epoch [5][200/313]	lr: 1.006e-04, eta: 7:45:30, time: 0.897, data_time: 0.001, memory: 66591, loss: 0.0058
Epoch [5][300/313]	lr: 1.099e-04, eta: 7:42:31, time: 0.891, data_time: 0.001, memory: 66591, loss: 0.0058
Epoch [6][100/313]	lr: 1.208e-04, eta: 7:41:02, time: 1.063, data_time: 0.152, memory: 66591, loss: 0.0058
Epoch [6][200/313]	lr: 1.309e-04, eta: 7:38:29, time: 0.898, data_time: 0.001, memory: 66591, loss: 0.0058
Epoch [6][300/313]	lr: 1.412e-04, eta: 7:35:59, time: 0.895, data_time: 0.001, memory: 66591, loss: 0.0058
```

**Comment by YuanLiuuuuuu** [comment]
Which dataset did you use?

**Comment by liming-ai** [comment]
> Which dataset did you use?

ImageNet-1k

**Comment by YuanLiuuuuuu** [comment]
It's mysterious. I re-trained mae on our machine recently, and the training log is the same as that on github. Maybe, you can re-train mae with the latest released version.

**Comment by liming-ai** [comment]
> It's mysterious. I re-trained mae on our machine recently, and the training log is the same as that on github. Maybe, you can re-train mae with the latest released version.

I re-train the MAE with latest released version, this bug is fixed, however, I still do not know the reason... Thanks for your answer.
